Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/WebElementsWrappers/PlayWrightWebElement.cs

```diff
@@ -28,10 +32,18 @@ public ILocator Locator
         {
             if (WebContext is null) throw new NullReferenceException("Please set web context.");
             // Locator for Playwright can be retrieved from Page element:
-            var selector = (ElementSelector.XpathSelector != null)
-                ? ElementSelector.XpathSelector
-                : ElementSelector.IdSelector;
-            return ((Page) WebContext.Page).GetPlaywrightPage().Locator(selector);
+            if (ElementSelector.XpathSelector != null)
```

Comment:
Is it a good idea to have selector validation here ? Does it mean that we will need to have it in every check ? Isn't it better to have different class and methods for getting Locators, which will check the selector inside them ?